rotkehlchen/exchanges/binance.py

```diff
@@ -995,6 +1110,8 @@ def _api_query_list_within_time_delta(
                 'startTime': from_ts,
                 'endTime': to_ts,
             }
+            if additional_options:
+                options = options | additional_options
```

Comment:
please don't use pipe operator with dicts.

(1) this is only in python 3.9, we utilize 3.7
(2) it's a bit ambiguous. You would have to know that its' equal to `update`. We try to be explicit in rotki so better use update.

Reply:
sure, wasnt aware of the python3.7 requirement, sorry for that